operations/security/product-security/product-vulnerability-process.md

```diff
@@ -0,0 +1,209 @@
+is an industry-wide accepted standard to determine the impact of vulnerabilities in software products and takes impacts 
+such as attack vector, complexity, user interaction, scope, and various other components into consideration.
+
+FIRST provides a CVSS calculator at the following URL: https://www.first.org/cvss/calculator/3.1
+
+The CVSS 3.1 provides a textural representation of the severity score in Section 5 of the Specification Document. The 
+mapping is included below:
+
+<table>
+  <thead>
+    <tr>
+      <th style="text-align:left">Rating</th>
+      <th style="text-align:left">CVSS Score</th>
+    </tr>
+  </thead>
+  <tbody>
+        <tr>
+            <td>None</td>
+            <td>0.0</td>
+        </tr>
+        <tr>
+            <td>Low</td>
+            <td>0.1 - 3.9</td>
+        </tr>
+        <tr>
+            <td>Medium</td>
+            <td>4.0 - 6.9</td>
+        </tr>
+        <tr>
+            <td>High</td>
+            <td>7.0 - 8.9</td>
+        </tr>
+        <tr>
+            <td>Critical</td>
+            <td>9.0 - 10.0</td>
+        </tr>
+  </tbody>
+</table>
+
+### Remediation SLAs
+
+(INTERNAL - TBD to Publish)
+
+### Issue Types
+Vulnerabilities must be assigned one of the following vulnerability types:
+- Authorization
+- Reducing Attack Surface
+- XSS
+- Input Validation
+- Phishing
+- Injection
+- Information Disclosure
+- DOS
+- Authentication
+- CSRF
+- Misconfiguration
+- 3rd Party Library
+- Remote Code Execution
+
+If no fitting category can be found, the security engineer must request the inclusion of a new type from the product 
+security team lead.
+
+### Backport Policy
+
+#### Vulnerabilities with Low Score to Medium Score:
+- Upcoming Release
+- Extended Support Release
+
+#### Vulnerabilities with High to Critical Score:
+- Upcoming Release
+- Extended Support Release
+- Last 3 Releases
+
+## Process Steps
+
+### Report
+Vulnerability reports are received from internal or external parties via one of the methods outlined above. The Product 
+Security Team must acknowledge to the reporter that it received the report within 24h on business days and will investigate 
+whether it’s a valid issue. If the vulnerability was published on a news site/blog/etc. and is considered a public leak 
+the marketing team should be notified to coordinate on the response in the respective Mattermost channel with a “@channel” mention.
+
+After the report was received, a “Security Vulnerability” ticket must be created in the Jira ticket that contains the 
+following information:
+
+* Title
+* Description
+* Category [If possible]
+* Reporter
+* Source
+
+The security engineer that acknowledges the initial report must create a Mattermost Playbook Run using the "Security 
+Vulnerability Playbook" and follow the process steps documented to continue with the Triage step. The playbook links to
+the Jira ticket created above, ensures the right process steps are completed, and informs relevant parties.
+
+###  Triage
```

Comment:
IMO, in most cases triage should happen before running kicking off the playbook run. Only for complex / difficult-to-repro cases the playbook may be necessary for triage.